### ISSUE TITLE
run RunPom error !

### ISSUE THREAD

**Issue by wonka80** [pierrebaque/EM#19]
IOError: [Errno 2] No such file or directory: './Potentials/Parts/Run_transfer/c0/2000.npy'
where is the file ?
thank you !

**Comment by pierrebaque** [comment]
Hello Wonka,

Did you run the RunParts script before?

Cheers
Pierre

**Comment by wonka80** [comment]
Hello Pierre,

I have run RunParts script  before, But ,failed.
the system output is ...........
WARNING (theano.sandbox.cuda): CUDA is installed, but device gpu3 is not available  (error: cuda unavailable).

My device is MacBook Pro without Nvidia device . 
So I change the theanorc file, set device = cpu.   .

Can you how to solve this problem ?
Thank you very much ! 
Best wishes ! 

--------------------------------------------
$ cat ~/.theanorc
[cuda]
root = /usr/local/cuda

[global]
device = cpu
floatX = float32
cxx = icpc
mode = FAST_RUN
openmp = True
openmp_elemwise_minsize = 10
[gcc]
cxxflags = -qopenmp -march=native -O3 -vec-report3 -fno-alias -opt-prefetch=2 -fp-trap=none
[blas]
ldflags = -lmkl_rt

**Comment by nagasanthoshp** [comment]
@pierrebaque can we implement it in a 'non-Nvidia" device?

**Comment by nagasanthoshp** [comment]
@pierrebaque  can we implement it in a 'non-Nvidia' device?

**Comment by nagasanthoshp** [comment]
@wonka80 Did you resolve the issue? If yes, please help me overcome mine. As I get the issue while running RunParts script. 
I use a CPU and non-Nvidia computer.

Thank you!

**Comment by wonka80** [comment]
@nagasanthoshp ,yes ,the problem has been solved,when we switch to a Nvidia computer .
Thank you very much !
Best wishes !
